frontend/components/common/CompatibleDevice.tsx

```diff
@@ -88,13 +79,18 @@ export function CompatibleDevice({
                      event.preventDefault();
                      event.stopPropagation();
                   },
-                  onKeyDown: (event) => {
-                     if ('Enter' === event.code) {
-                        event.preventDefault();
-                        event.stopPropagation();
-                     }
-                  },
                })}
+               sx={{
```

Comment:
Minor point but this seems like it could be on the summary element closer to where it's relevant.

Reply:
Wasn't sure how to do a css parent selector for the `[open]` attribute